app/models/katello/host/content_facet.rb

```diff
@@ -105,6 +113,7 @@ def calculate_and_import_applicability
       def import_applicability(partial = false)
```

Comment:
@ianballou is the `import_applicability` method relevant these days?

Reply:
It actually looks like it isn't... and the `katello:import_applicability` rake task seems to be out of date.  You'll get `uninitialized constant Katello::Host::ContentFacet::ApplicableContentHelper` if you run it.

In short, we can delete `def import_applicability` completely.  That could be in another PR if we don't wanna delay things any longer.

Reply:
Yeah definitely will handle it separately. Ty for confirming !

Reply:
I made an issue: https://projects.theforeman.org/issues/33554